sunbird_service_api_test/src/test/java/org/sunbird/integration/test/user/UnblockUserTest.java

```diff
@@ -111,11 +111,14 @@ public void testUnblockUnblockUserByUserIdFailure() {
         RESPONSE_JSON);
   }
 
-  @Test(dependsOnMethods = {"testUnblockUserSuccess"})
+  @Test()
   @CitrusTest
   public void testGetUnblockUserByUserIdSuccess() {
     getTestCase().setName(TEST_UNBLOCK_USER_GET_SUCCESS_WITH_VALID_USERID);
+    getAuthToken(this, true);
```

Comment:
Move following statements to beforeTest.

getAuthToken(this, true);
...
variable("userId", testContext.getVariable("userId"));
...

beforeTest() can take an argument to block or unblock user.